eventhub/function.yaml.template

```diff
@@ -0,0 +1,50 @@
+apiVersion: "nuclio.io/v1beta1"
+kind: "NuclioFunction"
+metadata:
+  name: eventhub
+spec:
+  description: >
+    An Azure Event Hub triggered function with a configuration that connects to an Azure Event Hub.
+    The function reads messages from two partitions, process the messages, invokes another function,
+    and sends the processed payload to another Azure Event Hub.
+
+  # Note that we're not specifying handler. This is because the Golang runtime
+  # can find handlers automatically.
+  runtime: "golang"
+  handler: main:SensorHandler
```

Comment:
But we are specifying the handler

Reply:
@TomerShor what do you mean? The name of handler in the code is `SensorHandler`

Reply:
That's fine, naming doesn't matter.
but the comment says "we're not specifying handler", and 2 rows below there is a specified handler:
```
handler: main:SensorHandler
```

Reply:
@TomerShor ah, now I noticed it :) will remove the comment